docs/QUICK-START.md

```diff
@@ -0,0 +1,388 @@
+        "facility": {
+          "facility_code": "onprem"
+        },
+        "instance": {
+          "id": "3f0c4d3d-00ef-4e46-983d-0e6b38da827a",
+          "hostname": "hw-a",
+          "network": {
+            "addresses": [
+              {
+                "address_family": 4,
+                "public": false,
+                "address": "192.168.1.105"
+              }
+            ]
+          },
+          "storage": {
+            "disks": [{"device": "/dev/sda"}]
+          }
+        },
+        "state": ""
+      },
+      "network": {
+        "interfaces": [
+          {
+            "dhcp": {
+              "arch": "x86_64",
+              "hostname": "hw-a",
+              "ip": {
+                "address": "192.168.1.105",
+                "gateway": "192.168.1.254",
+                "netmask": "255.255.255.0"
+              },
+              "mac": "a8:a1:59:66:42:89",
+      "name_servers": ["8.8.8.8"],
+              "uefi": true
+            },
+            "netboot": {
+              "allow_pxe": true,
+              "allow_workflow": true
+            }
+          }
+        ]
+      }
+    }
+    ```
+
+### Install and/or configure a Kubernetes cluster
+
+Cluster API requires an existing Kubernetes cluster accessible via kubectl. During the installation process the
+Kubernetes cluster will be transformed into a [management cluster] by installing the Cluster API [provider components], so it
+is recommended to keep it separated from any application workload.
+
+It is a common practice to create a temporary, local bootstrap cluster which is then used to provision
+a target [management cluster] on the selected [infrastructure provider].
+
+Choose one of the options below:
+
+1. **Existing Management Cluster**
+
+   For production use-cases a "real" Kubernetes cluster should be used with appropriate backup and DR policies and procedures in place. The Kubernetes cluster must be at least v1.19.1.
+
+   ```bash
+   export KUBECONFIG=<...>
+   ```
+
+2. **Kind**
+
+   **NOTE** [kind] is not designed for production use.
+
+   **Minimum [kind] supported version**: v0.9.0
+
+   [kind] can be used for creating a local Kubernetes cluster for development environments or for
+   the creation of a temporary [bootstrap cluster] used to provision a target [management cluster] on the selected infrastructure provider.
+
+   Create the kind cluster:
+   ```bash
+   kind create cluster
+   ```
+   Test to ensure the local kind cluster is ready:
+   ```
+   kubectl cluster-info
+   ```
+
+### Install clusterctl
+
+The clusterctl CLI tool handles the lifecycle of a Cluster API management cluster.
+
+#### Install clusterctl binary with curl on linux
+
+Download the latest release; on linux, type:
+```
+curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.0/clusterctl-linux-amd64 -o clusterctl
+```
+Make the clusterctl binary executable.
+```
+chmod +x ./clusterctl
+```
+Move the binary in to your PATH.
+```
```

Comment:
Feel free to ignore. ;)

Some markdown linting tools require a blank line before the opening backticks and after the closing backticks. No issues with how this looks on GitHub, but just thought I would mention it in case we ever decide to enforce markdown linting or we publish docs using a tool that is more strict.

Reply:
Thanks! I think I addressed lint issues related to spacing around backticks